google/cloud/vision_helpers/decorators.py

```diff
@@ -71,24 +71,28 @@ def _create_single_feature_method(feature):
     fx_doc += """
 
     Args:
-        image (:class:`~.{module}.types.Image`): The image to analyze.
+        image (:class:`~.{module}.Image`): The image to analyze.
         max_results (int):
             Number of results to return, does not apply for
             TEXT_DETECTION, DOCUMENT_TEXT_DETECTION, or CROP_HINTS.
         retry (int): Number of retries to do before giving up.
         timeout (int): Number of seconds before timing out.
+        metadata (Sequence[Tuple[str, str]]): Strings which should be
+            sent along with the request as metadata.
         kwargs (dict): Additional properties to be set on the
             :class:`~.{module}.types.AnnotateImageRequest`.
 
     Returns:
-        :class:`~.{module}.types.AnnotateImageResponse`: The API response.
+        :class:`~.{module}.AnnotateImageResponse`: The API response.
     """
 
     # Get the actual feature value to send.
-    feature_value = {"type": feature}
+    feature_value = {"type_": feature}
 
     # Define the function to be returned.
-    def inner(self, image, max_results=None, retry=None, timeout=None, **kwargs):
+    def inner(
+        self, image, max_results=None, retry=None, timeout=None, metadata=(), **kwargs
```

Comment:
I think retry, timeout, and metadata should also be keyword only arguments for consistency.

Reply:
https://github.com/googleapis/python-vision/blob/sijun/google/cloud/vision_helpers/decorators.py#L104

Reply:
I don't think that's a problem. If the signature is something like
```python
def inner(self, image, *, max_results=None, retry=None, timeout=None, metadata=(), **kwargs):
   ...
```
Then `kwargs` doesn't accidentally capture retry/timeout/metadata.
I think `max_results` should also be keyword only, but I'm less confidant about that.

Reply:
changed it to `def inner(self, image, *, max_results=None, retry=None, timeout=None, metadata=(), **kwargs)`